lib/access_import/accessors/accessor.rb

```diff
@@ -16,11 +16,17 @@ def add_other_accessors(accessors)
     end
   end
 
+  def add_records_hash
+    @records = hash_all unless @records
```

Comment:
What's the reason for the lazy initialization?

If you need it, the pattern for memoization is more like:

```ruby
def all
  @all ||= hash_all
end

def find(id)
  all[id.to_i]
end

# ... etc.
```

This way you don't need to add `add_records_hash` all over the place (which will likely be forgotten at one point and introduce a bug).